demo_keyframe_bundle_adjustment_meta/apps/main_program.cpp

```diff
@@ -0,0 +1,178 @@
+#include <iostream>
```

Comment:
Please add a gpl3 license file here with you as an author
Along 
```suggestion
/**
LIMO 2022
This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program.  If not, see <http://www.gnu.org/licenses/>.

\author Hujong Hu
**/

#include <iostream>
```